lib/scholars_archive/validators/graduation_year_validator.rb

```diff
@@ -0,0 +1,35 @@
+module ScholarsArchive::Validators
+  class GraduationYearValidator < ActiveModel::Validator
+    def validate(record)
+      return if record.graduation_year.empty? || valid_value(record)
+      add_error_message(record)
+      return
+    end
+
+    private
+
+    def valid_value(record)
+      grad_year = DateTime.strptime(record.graduation_year.to_i.to_s, "%Y").year
+      valid_date_string(record.graduation_year) && valid_date_range(grad_year) && valid_date_length(grad_year)
+    end
+
+    def valid_date_string(date)
+      return true if date.length == 4
+      false
+    end
+
+    def valid_date_range(grad_year)
+      return true if grad_year > 1868 && grad_year < (Date.today.year + 5)
```

Comment:
grad_year should be greater than or equal to 1868

Reply:
Oh oops hehe